amypet/backend_centiloid.py

```diff
@@ -392,6 +397,30 @@ def run(fpets, fmris, Cnt, tracer='pib', flip_pet=None, bias_corr=True,
                 fmsk: 100 * (suvr[fmsk] - CLA[fmsk][0]) / (CLA[fmsk][1] - CLA[fmsk][0])
                 for fmsk in fmasks if fmsk != 'ctx'}
 
+
+        # ---------------------------------
+        # > save csv with suvr and cl outputs
+        if csv_metrics == 'short':
+            csv_dict = {'path_outputs': out[onm]['opth'],
+                        **{f'suvr_{key}': value for key, value in out[onm]['suvr'].items()},
+                        **{f'cl_{key}': value for key, value in out[onm]['cl'].items() if key == 'wc'}}
+
+        if csv_metrics == 'long':
+            csv_dict = {'path_outputs': out[onm]['opth'],
+                        **{f'suv_{key}': value for key, value in out[onm]['avgvoi'].items()},
+                        **{f'suvr_{key}': value for key, value in out[onm]['suvr'].items()},
+                        **{f'suvr_pib_calc_{key}': value for key, value in
+                           out[onm]['suvr_pib_calc'].items()},
+                        **{f'suvr_pib_calc_transf_{key}': value for key, value in
+                           out[onm]['suvr_pib_calc_transf'].items()},
+                        **{f'cl_{key}': value for key, value in out[onm]['cl'].items()}}
+
```

Comment:
```suggestion
        else:
            raise KeyError(f"`csv_metrics` must be 'short' or 'long' (got {csv_metrics})")
```

Or maybe

```suggestion
        elif csv_metrics:
            raise KeyError(f"`csv_metrics`: unknown value ({csv_metrics})")

        if csv_dict:
```

(so `False`/`None` will skip CSV outputs)

Reply:
Yes better (probably option 2). @pjmark Do you have a preference? Could you please check this fits with the general format of your code?